app/controllers/contacts_controller.rb

```diff
@@ -69,6 +69,7 @@ def update_params
 
   def set_contact_by_email
     params.require(:email)
+    params[:email] = params[:email].downcase
```

Comment:
Could you actually create a separate method called downcase_email that runs before every every method that uses email?

Reply:
Sure

Reply:
Updated and added unit tests 😉